Mathlib/LinearAlgebra/Isomorphisms.lean

```diff
@@ -158,9 +158,11 @@ def quotientQuotientEquivQuotient : ((M ⧸ S) ⧸ T.map S.mkQ) ≃ₗ[R] M ⧸
   { quotientQuotientEquivQuotientAux S T h with
     toFun := quotientQuotientEquivQuotientAux S T h
     invFun := mapQ _ _ (mkQ S) (le_comap_map _ _)
-    left_inv := fun x => Quotient.inductionOn' x fun x => Quotient.inductionOn' x fun x =>
+    left_inv := fun x => Submodule.Quotient.induction_on _
+     x fun x => Submodule.Quotient.induction_on _ x fun x =>
       by simp [Quotient.mk''_eq_mk]
```

Comment:
Please check if `Quotient.mk''_eq_mk` is still used, here and below

Reply:
You're right it shouldn't be used any more, I'll change, thanks for the heads up!